### DISCUSSION TITLE
Plans for EDS integration

### DISCUSSION THREAD

**Discussion by ftaddesse** [nasa/cFS#689] [Q&A]
Hello, I recently started working on a project that involves cFS and I'm currently investigating the integration of EDS with cFE and other cFS applications.

According to the discussion on the thread [https://github.com/nasa/cFS/discussions/284](https://github.com/nasa/cFS/discussions/284), ESD is still work in progress. 

Are there any plans to include EDS integration in Caelum?

**Comment by dmccomas** [comment]
I can't speak to NASA's plans for EDS integration, but I would like to provide another cFS EDS reference/resource called [cFS Basecamp](https://github.com/cfs-tools/cfs-basecamp). I just added a more detailed response in discussion #284.
